augur/application/schema/alembic/versions/18_schedule_any_old_facade_repositories_to.py

```diff
@@ -0,0 +1,73 @@
+"""Schedule any old facade repositories to be re-cloned
+
+Revision ID: 18
+Revises: 17
+Create Date: 2023-05-02 17:35:18.891913
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+from sqlalchemy.sql import text
+import pathlib
+import shutil
+from augur.application.config import AugurConfig
+from augur.application.db.session import DatabaseSession
+import logging
+
+# revision identifiers, used by Alembic.
+revision = '18'
+down_revision = '17'
+branch_labels = None
+depends_on = None
+
+logger = logging.getLogger(__name__)
+
+
+def total_facade_reset():
+    conn = op.get_bind()
+
+    conn.execute(text(f"""
+
+    UPDATE augur_operations.collection_status
+    SET facade_status='Pending', facade_task_id=NULL, facade_weight=NULL,commit_sum=NULL,facade_data_last_collected=NULL;
+
+    UPDATE repo
+    SET repo_path=NULL,repo_name=NULL;
+    """))
+
```

Comment:
After thinking on this, I think it would be better if we just wrote a function in this alembic script that creates the new path with the repo id and path identifier. Then used a mv command to move the clone to the new directory. We should be able to do this because we already have the location of the facade directory, and we can tell which directories are old because they are directories that are just an integer (repo_group_id) rather than the new format which is an integer-string (repo_id-repo_path). I think this would make more sense to existing users, and all the facade repos wouldn't have to go back through the process of moving through statues (this isn't a huge deal it just clogs up facade for new repos until of these reset ones make it through which isn't ideal)

Reply:
I didn't want to do this because we want to calculate the weight for old repos anyways and its not that much more to ask to add the cloning overhead to that weight calculation since it happens in the clone_repos method. Also depending on the configuration that people have on their instances moving each repo could take a long time which is unexpected for an alembic script.

Reply:
@ABrain7710 : So, what do you think?

Reply:
@IsaacMilarky / @sgoggins What Isaac mentioned makes sense. If we are taking this approach with old repos for facade I think we should also update the core and secondary tasks so we ensure they compute the issue_pr_sum. To make sure we don't reset every repo on padres though we should only update core and secondary tasks to pending if the issue_pr_sum is NULL.